### PR TITLE
[PREVIEW] Site re-brand + Topics Integration Branch

### PR DESCRIPTION
**Pull request by wesbos**
All design between @adelasofia and I will be in this branch. 

Rebase early and rebase often.

### CODE REVIEW

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
This seems to be failing intermittently when pushing to 'dcp2-searchisko.rhcloud.com'. We should re-asses this after the switch back to the Docker nightly instance of DCP has been made. This should be done in the next 30m.

**Comment by paulrobinson** [comment]
As we might expect, it's breaking the Acc tests, due to the nav change. See: http://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/developers.redhat.com-frontend-acceptance-tests/1070/console

This will need fixing before we go live.

**Comment by wesbos** [comment]
@paulrobinson You said you were only seeing the mobile nav for this page? Preview seems to be working fine

**Comment by paulrobinson** [comment]
This is what i see: https://www.dropbox.com/s/a469zgyjc7fvk3o/Screenshot%202016-02-01%2018.32.12.png?dl=0 on http://it-developers-pr.stage.redhat.com/pr/715/build/587/

**Comment by wesbos** [comment]
@paulrobinson The Flexbox issue with the layout is now fixed

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
@wesbos @paulrobinson @RubyGemTSL 
I believe this is failing due to acceptances tests legitimately. I'm not going to investigate further.

**Comment by paulrobinson** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
@paulrobinson the tests will continue to fail for this PR as the nav menu has changed. The tests need updating.

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
docker build

File: Gemfile.lock [hunk 1]
```diff
@@ -424,6 +424,5 @@ DEPENDENCIES
   therubyracer
   uglifier (~> 2.0.1)
   wraith (~> 1.3.0)
-
```
Comment (ollyjshaw) [inline Gemfile.lock]: You'll need to revert this. It will cause issues with the docker PR builds and the live build

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
This seems to work on my machine. Although it needs another rebase :-(

I'll merge it once the rebase is done and it passes CI tests.

**Comment by wesbos** [comment]
Closing this - will reopen a new one.

**Pull request closed without merging**